### PR TITLE
miner: add logs for displaying fees change

### PR DESCRIPTION
**Pull request by rjl493456442**

### CODE REVIEW

File: miner/payload_building.go [hunk 1]
```diff
@@ -77,6 +79,10 @@ func (payload *Payload) update(block *types.Block, fees *big.Int) {
 	if payload.full == nil || fees.Cmp(payload.fullFees) > 0 {
 		payload.full = block
 		payload.fullFees = fees
+
+		feesInEther := new(big.Float).Quo(new(big.Float).SetInt(fees), new(big.Float).SetInt(big.NewInt(params.Ether)))
```
Comment (holiman) [inline miner/payload_building.go:83]:
```suggestion
		feesInEther := new(big.Float).Quo(new(big.Float).SetInt(fees), big.NewFloat(params.Ether))
```

**Comment by rjl493456442** [comment]
todo: fix the mutex stuff in this PR

**Review by holiman — APPROVED** [review]
Still LGTM

**Review by MariusVanDerWijden — APPROVED** [review]
LGTM

**Pull request merged**